### DISCUSSION TITLE
tailwindcss 3.4.3 ?

### DISCUSSION THREAD

**Discussion by blondie63** [cdnjs/cdnjs#14248] [General]
I've seen that latest tailwindcss here is 2.2.19
why no  more updates ?

**Comment by songlim327** [comment]
In tailwindcss version 3 and above, it is recommended to use PostCSS plugin or Tailwind CLI to compile your CSS, and then upload the compiled CSS to your production server. See more [Tailwindcss Installation Documentation](https://tailwindcss.com/docs/installation)
